pkg/shares/compact_shares_test.go

```diff
@@ -81,7 +81,7 @@ func Test_processCompactShares(t *testing.T) {
 		{"single big tx", appconsts.ContinuationCompactShareContentSize * 4, 1},
 		{"many big txs", appconsts.ContinuationCompactShareContentSize * 4, 10},
 		{"single exact size tx", exactTxShareSize, 1},
-		{"many exact size txs", exactTxShareSize, 10},
+		{"many exact size txs", exactTxShareSize, 100},
```

Comment:
[for reviewers] this surfaced a bug in the implementation of this PR that a smaller txSize didn't surface so I opted to increase the number of transactions in this test